<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogRunAdvanced</class>
 <widget class="QDialog" name="DialogRunAdvanced">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Run Script(s)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="toolbar_background" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>56</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QWidget" name="nav_bar" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="left">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>34</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_button_add_script">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add a script to the list to execute. Parameters can be specified by click on the &amp;quot;Arguments&amp;quot; column in the table below.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Add Script</string>
           </property>
           <property name="icon">
            <iconset resource="../../../register_browser.qrc">
             <normaloff>:/images/resources/icon_button_add_alternate.png</normaloff>:/images/resources/icon_button_add_alternate.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_button_remove_script">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remove a script from the execution list.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Remove Script</string>
           </property>
           <property name="icon">
            <iconset resource="../../../register_browser.qrc">
             <normaloff>:/images/resources/icon_button_remove_alternate.png</normaloff>:/images/resources/icon_button_remove_alternate.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_button_move_up">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Move a script up in the execution order so that it run earlier.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Move Up</string>
           </property>
           <property name="icon">
            <iconset resource="../../../register_browser.qrc">
             <normaloff>:/images/resources/icon_move_up.png</normaloff>:/images/resources/icon_move_up.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_button_move_down">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Move a script down in the execution order so that it is run later.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Move Down</string>
           </property>
           <property name="icon">
            <iconset resource="../../../register_browser.qrc">
             <normaloff>:/images/resources/icon_move_down.png</normaloff>:/images/resources/icon_move_down.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_button_save_suite">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save the suite of scripts to an external file that can be re-run later.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Save Suite</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_button_load_suite">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Load a previously saved suite of scripts so that it can be re-run.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Load Suite</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="right">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>34</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>166</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;text-align:center;background-color:white;

border-radius: 8px;
border:0px solid #ccc;
</string>
     </property>
     <property name="text">
      <string>This dialog allows you to run a set of scripts and capture the output. Double click on a column to set arguments, save the output, or capture a register dump.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="m_tree_script_list">
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Script</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Arguments</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Output</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Registers</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>370</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="m_check_close_scripts_on_exit">
        <property name="text">
         <string>Close Scripts on Exit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_button_run">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_button_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../register_browser.qrc"/>
  <include location="../../../register_browser.qrc"/>
  <include location="../../../register_browser.qrc"/>
  <include location="../../../register_browser.qrc"/>
 </resources>
 <connections/>
</ui>
